### DISCUSSION TITLE
Does this plugin expose or can expose zigbee2mqtt scenes to homebridge?

### DISCUSSION THREAD

**Discussion by anjos** [itavero/homebridge-z2m#829] [Q&A]
If not, would anyone know what would be the best way to recall a zigbee2mqtt scene via homebridge?  Thanks a lot!

**Comment by itavero** [comment]
It currently does not do this. I haven't looked into this myself as I make most scenes in HomeKit.

If you have a suggestion on how we could expose them, let me know.

In the mean time you could use homebridge-mqttthing

**Comment by anjos** [comment]
I come from deCONZ - scenes stored in software are exposed to HomeKit via their own plugin.  [Here is the original issue/discussion](https://github.com/ebaauw/homebridge-hue/issues/475) (back from 2019) with the rationale and intent.

> Note on ["homebridge-hue" plugin](https://github.com/ebaauw/homebridge-hue): A [new implementation](https://github.com/ebaauw/homebridge-deconz) is now overtaking that plugin, and promises to be more dynamic.

One issue that was discussed then was how to implement this reasonably - at the time, HomeKit did not provide a software-equivalent of a spring-loaded switch.  In this regard, you can probably retrieve elements from its UI and details that may be relevant to z2m integration.

I remain available to test anything you may propose locally, or to interact on this.

**Reply by itavero** [reply]
Ah, he seems to have used a custom characteristic. Pretty smart as that probably won't interfere with "regular" voice commands and such.

**Reply by anjos** [reply]
BTW I checked homebridge-mqttthing - there isn't an accessory type in there that immediately looks like a z2m "scene".  Any particular suggestions?

**Reply by itavero** [reply]
Not really. The only thing I can think of is using a "switch"
